radium-blockchain-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,233 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"adminUsername": {
+			"type": "string",
+			"metadata": {
+				"description": "Username for the Virtual Machine."
+			}
+		},
+		"adminPassword": {
+			"type": "securestring",
+			"metadata": {
+				"description": "Password for the Virtual Machine."
+			}
+		},
+		"dnsLabelPrefix": {
+			"type": "string",
+			"metadata": {
+				"description": "Hostname assigned to the Public IP, also used as the VM Name. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+			}
+		},
+		"installMethod": {
+			"type": "string",
+			"defaultValue": "From_Source",
+			"allowedValues": [
+				"From_Source"
```

Comment:
Is there way to install this from a binary? Building from source can take a lot of time and has the user paying for compute time